vowpalwabbit/array_parameters.h

```diff
@@ -101,7 +101,7 @@ class sparse_parameters
 
   weight* first()
   {
-    THROW_OR_RETURN_VOID("Allreduce currently not supported in sparse");
+    THROW_OR_RETURN_NORMAL("Allreduce currently not supported in sparse", nullptr);
```

Comment:
Is there a THROW_OR_RETURN() that functions differently you are avoiding colliding with?

Reply:
Whoops, yes there is a macro that handles both these cases. Using that now.